java/src/main/java/com/google/cloud/dataproc/templates/gcs/README.md

```diff
@@ -34,3 +34,23 @@ bin/start.sh \
 --templateProperty gcs.spanner.output.saveMode=<Append|Overwrite|ErrorIfExists|Ignore> \
 --templateProperty gcs.spanner.output.primaryKey=<column[(,column)*] - primary key columns needed when creating the table>
 ```
+
+
+## 3. GCS to JDBC
```

Comment:
Readme should specifically call out that JDBC driver needs to be downloaded separately and then uploaded to GCS location.

Reply:
Done.